frontend/package.json

```diff
@@ -3,8 +3,8 @@
   "version": "0.1.0",
   "private": true,
   "engines": {
```

Comment:
It's currently not there but should we also define the node and npm versions in the `cypress/package.json` as well?

Reply:
I don't think we need to add it anywhere else since those others technically don't rely on a specific version of node.

Reply:
Oh ok! Got it! 👍